system_test/run.py

```diff
@@ -202,7 +210,7 @@ def test_basic_consuming(self):
         ok, line = dumpServerStdIO(s3connect, "Wait for connect to process and commit",
                                   until="Successfully uploaded chunk for system-test-0",
                                   until_fail="ERROR",
-                                  timeout=5, trim_indented=True)
+                                  timeout=15, trim_indented=True)
```

Comment:
I could never get these tests to pass until I upped the timeout, and even then it's a little iffy.